Godeps/_workspace/src/github.com/GoogleCloudPlatform/kubernetes/pkg/api/v1/conversion_generated.go

```diff
@@ -1564,6 +1564,11 @@ func convert_api_PodSpec_To_v1_PodSpec(in *api.PodSpec, out *PodSpec, s conversi
 	} else {
 		out.ImagePullSecrets = nil
 	}
+
+	// Carry conversion
+	out.DeprecatedServiceAccount = in.ServiceAccountName
+	out.DeprecatedHost = in.NodeName
```

Comment:
This is only here because I'm assuming we have to round-trip back to the deprecated field... @smarterclayton, is that true?

Also, I hate modifying a generated file but I couldn't find a cleaner place to put it